commands/scale_cluster.go

```diff
@@ -316,7 +330,7 @@ func scaleClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 
 	// Ask for confirmation for the scaling action.
 	if !cmdForce {
-		err = confirmScaleCluster(args, currentScalingMax, currentScalingMin, desiredCapacity)
+		err = confirmScaleCluster(args, currentScalingMax, currentScalingMin, statusWorkers)
```

Comment:
I think the wording with current and desired is a good thing. Also having the same semantics when providing and argument and using it within a function. Now `statusWorkers` is given and in the function it is called `currentWorkers`. That is all pretty confusing and does not make it better.